packages/financial-templates-lib/src/price-feed/UniswapPriceFeed.js

```diff
@@ -114,6 +141,20 @@ class UniswapPriceFeed extends PriceFeedInterface {
       return;
     }
 
+    // Filter out events where price is null.
+    this.events = events.filter(e => e.price !== null);
+
+    // Price at the end of the most recent block.
+    this.lastBlockPrice = this.events[this.events.length - 1].price;
+
+    // Compute TWAP up to the current time.
+    this.currentTwap = this._computeTwap(this.events, currentTime - this.twapLength, currentTime);
+
+    this.lastUpdateTime = currentTime;
+  }
+
+  async _getSortedSyncEvents(startBlock, endBlock) {
+    const events = await this.uniswap.getPastEvents("Sync", { fromBlock: startBlock, toBlock: endBlock });
```

Comment:
tiny nit: it might be cleaner to just refer to `fromBlock` and `toBlock` through out this file so we don't need to have this translation between `fromBlock: startBlock` & `toBlock: endBlock`. feel free to ignore this comment but it just seems that it would be more consistent to always use `fromBlock` and `toBlock` notation for these vars.

Reply:
Agreed. Will change.